pkgs/by-name/lh/lha/package.nix

```diff
@@ -7,27 +7,30 @@
 
 stdenv.mkDerivation {
   pname = "lha";
-  version = "unstable-2021-01-07";
+  version = "unstable-2024-11-27";
 
   src = fetchFromGitHub {
     owner = "jca02266";
     repo = "lha";
-    rev = "03475355bc6311f7f816ea9a88fb34a0029d975b";
-    sha256 = "18w2x0g5yq89yxkxh1fmb05lz4hw7a3b4jmkk95gvh11mwbbr5lm";
+    rev = "26b71be85a762098bdeb95f4533045c7dad86f31";
+    hash = "sha256-jiYTBqDXvxTdrvHYaK+1eo4xIpl+B9ZljhBBYD0BGzQ=";
   };
 
   nativeBuildInputs = [ autoreconfHook ];
 
-  meta = with lib; {
+  meta = {
```

Comment:
Could you add `meta.homepage`?
https://github.com/jca02266/lha

Reply:
Yes, added this link to the `meta.homepage`.